### PR TITLE
Run check nits

### PR DESCRIPTION
**Pull request by johndoe31415**
Some minor fixes to run-checker.sh

### CODE REVIEW

File: run-checker/run-checker.sh [hunk 1]
```diff
@@ -67,7 +72,7 @@ rkill () {
         done
     fi
     if [ "$pid" != "$notpid" ]; then
-	kill -s "$signal" "$pid"
+    kill -s "$signal" "$pid"
```
Comment (levitte) [inline run-checker/run-checker.sh]: This is not right

Comment (johndoe31415) [inline run-checker/run-checker.sh]: You're correct, don't know what happened there. Fixed in  e374795.

**Comment by levitte** [comment]
Merging now.

**Comment by levitte** [comment]
Merged and installed.

Thank you!

**Pull request closed without merging**